src/SpineRuntime/Animation.js

```diff
@@ -0,0 +1,70 @@
+var spine = require('../SpineRuntime') || {};
+spine.Animation = function (name, timelines, duration)
```

Comment:
For now, my opinion is to keep the structure the same. 

Many of the classes, are written such that spine.SomeClass/SomeProperty is used all over the place.

The update for this would require substantial sub-structure changes, and so lots of additional testing to ensure changes function properly.

As such the top line of ```var spine = require('../SpineRuntime') || {};``` is necessary for the module type configuration for the repo.

However, I agree, aside from cjs's pattern, it's much more standard to leave out the parent object association.